### DISCUSSION TITLE
Returning result when navigate

### DISCUSSION THREAD

**Discussion by Hoang-Nguyenn** [csells/go_router#311] [General]
Hi team, do we have returning result when navigate like Navigation 1.0 ? 
Example : final bool canDelete = await Navigator.pushNamed("confirmScreen");
Navigator.pop(context,true);

**Comment by csells** [comment]
Check out these docs: https://gorouter.dev/user-input

**Reply by Hoang-Nguyenn** [reply]
for example. If i have screen A and screen B.
I want handle some api calling on screen A after screen B pop, how i can do that on screen A ? Your way seem like you add an object into global list

**Reply by csells** [reply]
Check out these docs: gorouter.dev/user-input
